g2p/__init__.py

```diff
@@ -21,7 +21,13 @@
 if sys.stderr.encoding != 'utf8' and hasattr(sys.stderr, 'buffer'):
     sys.stderr = io.TextIOWrapper(sys.stderr.buffer, encoding="utf8")
 
+
+_g2p_cache = {}
+
 def make_g2p(in_lang: str, out_lang: str, tok_lang=None):
+    if (in_lang, out_lang, tok_lang) in _g2p_cache:
```

Comment:
Yes, it makes a fair bit of difference during unit testing. I didn't test with `readalongs align`, but I expect it will make a difference there too, since we call `make_g2p()` for each word.